src/WebJobs.Script.WebHost/Controllers/KeysController.cs

```diff
@@ -260,5 +264,15 @@ private bool IsFunction(string functionName)
             string functionPath = Path.Combine(_settings.ScriptPath, functionName);
             return ScriptHost.TryReadFunctionConfig(functionPath, out json, _fileSystem);
         }
+
+        private void LogRequest()
+        {
+            if (Request != null)
```

Comment:
This type of logging is a cross-cutting concern, and now handled by the HTTP tracing that was re-added. so not needed here.